.eslintrc.js

```diff
@@ -168,6 +184,12 @@ module.exports = {
         "jsdoc/require-returns": "off",
         "handsontable/restricted-module-imports": "off",
       }
-    }
+    },
+    {
+      "files": ["scripts/**"],
+      "rules": {
```

Comment:
It's a great idea. I was thinking the same. I'd extend the list of rules. It's common that the script uses globals such as `console.log` or using native ES syntax (for of). 

We can clean up files within `./scripts` from inline comments that disable rules and add them to the list. WDYT?